modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -55,10 +55,31 @@ function inope_policy_review_tab_title_callback($node) {
  *   policy nodes.
  */
 function inope_policy_review_page($node) {
-  // Create a build array to be displayed on the page.
-  $build = array(
-    'inope_policy_review_form' => drupal_get_form('inope_policy_review_form', $node),
-  );
+  global $user;
+  $account = $user;
+  $build = array();
+
+  // TODO: Ask client if they want previously submitted approval to be allowed
+  // to be overwritten, this is the current functionality.
+  $results = inope_policy_review_select_query($node, $account, 0);
+
+  // TODO: When the form is rebuilt query above this doesn't pick up the new
+  // entry.
+  // If the active user has previously submitted something only display a
```

Comment:
Unless the problem wasn't clear, which is a possibility, this is telling me that this branch wasn't functionally tested. That is a real no-no. So I am hoping it was the former, but if it was the later, each PR needs to be functionally tested by you for the business requirements. It's kind of assumed that this has happened -- so that a proper code review can take place.  Then a thorough functional code review will happen on the code reviewers side.  Again that second one assumes that you've already had a thorough functional on your side.